src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  hasLoadingError: boolean,
+  setHasLoadingError: (arg: boolean) => void,
+  isAddingErrorShown: boolean,
+  setIsAddingErrorShown: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+export const ErrorNotification = ({
+  hasLoadingError,
+  setHasLoadingError,
+  isAddingErrorShown,
+  setIsAddingErrorShown,
+}: Props) => {
+  const [isClosePressed, setIsCLosePressed] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setHasLoadingError(false);
+      setIsAddingErrorShown(false);
+    }, 3000);
+  }, [isAddingErrorShown]);
```

Comment:
It's a good practice to clear timeouts on unmount.
To do it return anonymous function inside useEffect and clearTimeout.